hive-llap/README.md

```diff
@@ -0,0 +1,125 @@
+
+# Hive LLAP Initialization Action
+This initial action configures Google Cloud Dataproc to run Hive LLAP. For more information on Hive LLAP please visit: https://cwiki.apache.org/confluence/display/Hive/LLAP
+
+## Using this initialization action
+
+**:warning: NOTICE:** See [best practices](/README.md#how-initialization-actions-are-used) of using initialization actions in production.
+
+You can use this initialization action to create a new Dataproc cluster with LLAP enabled:
+
+1. Use the `gcloud` command to create a new cluster with this initialization action. By default, LLAP will consume all nodes but one. If you want to customize the number
+of nodes LLAP runs on, please use the metadata paramater, num-llap-nodes to specifiy how many nodes to deploy LLAP on. You need at least 1 node open to run Tez AM's. You can also include the use of the 'init-actions-repo' metadata option. This will auto download the additonal files in the repo including a file for restarting LLAP (llap_restart.sh).
```

Comment:
Needs better explanation that we download the `llap_start.sh` from a default gcs bucket, and the user can pass a custom gcs bucket to download it from.

Reply:
added more language here